### PR TITLE
CNV-20962: SELinux updates, part 1

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.10+ (possibly earlier?)

Issue: [CNV-20962](https://issues.redhat.com//browse/CNV-20962)

Link to docs preview: https://54597--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-additional-security-privileges-controller-and-launcher.html#virt-extended-selinux-policies-for-virt-launcher_virt-additional-security-privileges-controller-and-launcher

QE review:
- [x] QE has approved this change.

Additional information: I plan to open a small PR to add another SELinux policy that is only in 4.12. I will also open a separate PR for the SCC fixes that are mentioned in the Jira.

### CODE REVIEW

File: modules/virt-extended-selinux-policies-for-virt-launcher.adoc [hunk 1]
```diff
@@ -6,18 +6,32 @@
...
+** `allow process self (netlink_audit_socket (nlmsg_relay))`
+
+* The following policies allow `libvirtd` to access `hugetblfs`, which is required to support huge pages:
+
+** `allow process hugetlbfs_t (dir (add_name create write remove_name rmdir setattr))` 
+** `allow process hugetlbfs_t (file (create unlink))`
++
+[NOTE]
+====
+Huge pages are a method of managing large amounts of memory by increasing the memory page size.
```
Comment (jean-edouard) [inline modules/virt-extended-selinux-policies-for-virt-launcher.adoc:30]: How about a pointer to the huge pages section instead?
(at `virt/virtual_machines/advanced_vm_management/virt-using-huge-pages-with-vms.html`)
This one line might be a bit of an over-simplification.

**Review by jean-edouard — COMMENTED** [review]
Looks good!
Just a couple suggestions in-line.

**Review by jean-edouard — APPROVED** [review]

**Comment by kbidarkar** [comment]
LGTM

**Review by lpettyjo — COMMENTED** [review]
You mention "Version(s): 4.10+ (possibly earlier?)", but you don't have an enterprise-4.10 label. Please add if required, or let me know if you need me to add it.

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

**Pull request merged**